Task_DEV_5/Task_DEV_5/SecondSorter.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Text;
+using System.IO;
+
+namespace task_DEV_5
+{
+    class SecondTextSort
+    {
+        private string path { get; set; }
+        public FileStream fstream;
+
+        public SecondTextSort(string path)
+        {
+            this.path = path;
+        }
+
+        public string ReadLine()
+        {
+            StringBuilder builder = new StringBuilder();
+            char symbol;
+            while ((symbol = (char)fstream.ReadByte()) != '\n')
+            {
+                if (fstream.Length == fstream.Position)
```

Comment:
You can move this condition to the while condition using or (||).

Reply:
Done.